baseStats.json

```diff
@@ -321,13 +321,12 @@
       "evolvesTo": 0,
       "evolvesFrom": "Kakuna",
       "quickMoves": [
-        "Bug Bite",
-        "Poison Jab"
+        201,
+        224
       ],
       "cinematicMoves": [
-        "Sludge Bomb",
-        "Aerial Ace",
-        "X Scissor"
+        90,
+        45
```

Comment:
Nah, lemme explain, the change was that we used to be mapping the move by name. While Pogo gives us the move by id. So i replaced all the moves (through a script) from by-name to by-id. The ordering doesn't matter. If you notice in the other's you mentioned there was X Scissor, and there isn't the ID 100.